noggin/security/ipa.py

```diff
@@ -97,6 +98,35 @@ def otptoken_del(self, ipatokenuniqueid):
         data = self._request('otptoken_del', [], params)
         return data['result']
 
+    def otptoken_sync(self, user, password, first_code, second_code, token=None):
+        """
+        Sync an otptoken for a user.
+
+        :param user: the user to sync the token for
+        :type user: string
+        :param password: the user's password
+        :type password: string
+        :param first_code: the first OTP token
+        :type first_code: string
+        :param second_code: the second OTP token
+        :type second_code: string
+        :param token: the token description (optional)
+        :type token: string
+        """
+        data = {
+            'user': user,
+            'password': password,
+            'first_code': first_code,
+            'second_code': second_code,
+            'token': token,
+        }
+        try:
+            url = "https://" + self._host + "/ipa/session/sync_token"
+            response = requests.post(url=url, data=data, verify=self._verify_ssl)
```

Comment:
Any reason why you're not using `self._session` here instead of `requests` directly?

Reply:
Changed this now.